inc/classes/busting/class-facebook-sdk.php

```diff
@@ -100,6 +100,15 @@ public function replace_url( $html ) {
 
 		$html      = str_replace( $tag, $replace_tag, $html );
 		$file_path = $this->get_busting_file_path( $locale );
+		$xfbml     = $this->get_xfbml_from_url( $tag ); // Default value should be set to false.
+		$app_id    = $this->get_appId_from_url( $tag ); // APP_ID is the only required value.
+		$version   = false === $this->get_version_from_url( $tag ) ? 'v5.0' : $this->get_version_from_url( $tag ); // If version is not available set it to the latest: v.5.0.
```

Comment:
Small enhancement: first assign the return of `$this->get_version_from_url( $tag )` to a variable, and then do the ternary operator check with that variable. That way you do only one method call, not 2